locust/runners.py

```diff
@@ -905,7 +906,6 @@ def client_listener(self):
                 self.connection_broken = True
                 gevent.sleep(FALLBACK_INTERVAL)
                 continue
-            self.connection_broken = False
```

Comment:
I dont really know this code. Can you ping the person who wrote it and see if s/he thinks this makes sense?

Reply:
Hi @delulu what do you think about it?